ChairManager/main.cpp

```diff
@@ -5,11 +5,20 @@
 #include <windows.h>
 #include <cstdio>
 #include "UI.h"
+#include <gtest/gtest.h>
 
 int APIENTRY _tWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPTSTR lpCmdLine, int nCmdShow) {
```

Comment:
`return 0` is missing somewhere (see compiler warning)

Reply:
fixed